portfolio/src/main/webapp/script.js

```diff
@@ -29,9 +29,19 @@ function addRandomGreeting() {
   greetingContainer.innerText = greeting;
 }
 
-function getComments() {
-  fetch('/data').then(response => response.json()).then((comments) => {
-    const documentElement = document.getElementById('comment-data');
-    documentElement.innerText = comments;
-  });
+function requestTranslation() {
+  const text = document.getElementById('text').value;
```

Comment:
Maybe call this comment or message. Since the returned value is translatedMessage, maybe this should be message.

Reply:
done